qmlti/src/resources/PerceptionSoap.php

```diff
@@ -317,6 +316,72 @@ public function get_access_assessment_notify($assessment_id, $participant_name,
     return $access_assessment;
   }
 
+  public function create_schedule_participant($schedule_id, $schedule_name, $assessment_id, $participant_id, $restrict_times = TRUE, $schedule_starts, $schedule_stops, $group_id, $group_tree_id, $web_delivery, $restrict_attempts, $max_attempts, $monitored, $test_center_id, $min_days_between_attempts, $time_limit_override, $time_limit, $offline_delivery) {
```

Comment:
Missing PHPDoc. Probably a good idea if someone else has to maintain this in the future.